io_ogre/meshy.py

```diff
@@ -89,10 +89,13 @@ def execute(self, context):
 
         if merged: context.scene.objects.unlink( merged )
 
-        if sys.platform.startswith('linux') or sys.platform.startswith('darwin') or sys.platform.startswith('freebsd'):
-            subprocess.Popen([CONFIG['MESH_PREVIEWER'], path+'/preview.mesh'])
-        else:
-            subprocess.Popen( [CONFIG['MESH_PREVIEWER'], 'C:\\tmp\\preview.mesh'] )
+        try:
+            if sys.platform.startswith('linux') or sys.platform.startswith('darwin') or sys.platform.startswith('freebsd'):
+                subprocess.Popen([CONFIG['MESH_PREVIEWER'], path + '/preview.mesh'])
+            else:
+                subprocess.Popen([CONFIG['MESH_PREVIEWER'], 'C:\\tmp\\preview.mesh'] )
+        except:
+            Report.messages.append('ERROR: Ogre Meshy not found at "%s"' % CONFIG['MESH_PREVIEWER'])
```

Comment:
```suggestion
            Report.messages.append('ERROR: Viewer not found at "%s"' % CONFIG['MESH_PREVIEWER'])
```
it does not have to be Meshy

Reply:
Right, I think it mentions meshy in other places but its true.